centreon/www/class/config-generate/engine.class.php

```diff
@@ -322,6 +323,21 @@ private function setLoggerCfg(): void
         }
     }
 
+    /**
+     * This method indicates if engine notifications needs to be disabled.
+     * conditions: "notification" feature flag AND isCloudPlatform must be true
+     *
+     * @return bool
+     */
+    private function shouldEngineNotificationsBeDisabled(): bool
+    {
+        $kernel = \App\Kernel::createForWeb();
+        $featureFlags = $kernel->getContainer()->get(Core\Common\Infrastructure\FeatureFlags::class);
+        $flags = $featureFlags->getAll();
+
+        return ($flags['notification'] === true && $featureFlags->isCloudPlatform() === true);
```

Comment:
```suggestion

        return ($featureFlags->isEnabled('notification') === true && $featureFlags->isCloudPlatform() === true);
```

Reply:
Damn I've missed this method... thanks :)